src/Manifest/ManifestManager.php

```diff
@@ -64,23 +67,62 @@ public function writeFileManifest(
      * @param string $fileName
      * @param string $destination
      * @param string[] $primaryKeyColumns
+     * @param string $delimiter
+     * @param string $enclosure
+     * @param string[] $columns
+     * @param bool $incremental
+     * @param mixed[][] $metadata
+     * @param mixed[][] $columnMetadata
      */
     public function writeTableManifest(
         string $fileName,
         string $destination = '',
-        array $primaryKeyColumns = []
+        array $primaryKeyColumns = [],
+        string $delimiter = ',',
+        string $enclosure = '"',
+        array $columns = [],
+        bool $incremental = false,
+        array $metadata = [],
+        array $columnMetadata = []
```

Comment:
asi bych dal delimiter a enclosure nakonec ? 
a nerikali jsme, ze se to udela pres pole? aby se nemuselo zadavat tolik parametru

Reply:
Oba use-case jsou platné IMO. Když potřebuješ jen některé, tak si musíš pamatovat, které to jsou. Plus jsem přidal extra metodu na array pod tím ;)

Reply:
Hodit na konec je můžu.

Reply:
aha, jasne, ja koukal na nejakej castecnej diff :-/ cool